README.md

```diff
@@ -1,7 +1,7 @@
 ### Team Dev Client
 
-Client repository for team dev project.
-
+conflicting changes
```

Comment:
Is this change still needed? I would suspect you'd want to keep the original text?